SDWebImageAVIFCoder/Classes/Private/ColorSpace.h

```diff
@@ -0,0 +1,19 @@
+//
+//  ColorSpace.h
+//  SDWebImageAVIFCoder
+//
+//  Created by psi on 2020/03/15.
+//
```

Comment:
These comment can use other files template. But you can leave your name here :) (Not the `DreamPiggy`)

Reply:
OK, I wrote my real name!